pyproject.toml

```diff
@@ -615,8 +615,6 @@ filterwarnings = [
   "ignore:unclosed <socket.socket",
   # Remove this in a future release of PySpark.
   "ignore:distutils Version classes are deprecated. Use packaging.version instead.",
-  "ignore:datetime.datetime.utcnow\\(\\) is deprecated and scheduled for removal in a future version.",
```

Comment:
huh, not sure where this went, theres no references in the codebase
https://github.com/search?q=repo%3Aapache%2Ficeberg-python%20utcnow&type=code

Reply:
it might be used in one of the dependency packages, but we don't need to filter the warning in that case